lib/functionsEmulator.js

```diff
@@ -45,6 +45,23 @@ var _pollOperation = function(op, controller) {
   });
 };
 
+var _setIdleFunctionsIntervals = function() {
```

Comment:
should name it _getIdleFunctionsIntervals, since it's not actually doing any setting